src/sentry/api/endpoints/organization_details.py

```diff
@@ -137,6 +137,12 @@
         bool,
         org_serializers.GITHUB_PR_BOT_DEFAULT,
     ),
+    (
+        "githubOpenPRComments",
```

Comment:
hmm i feel like this name is pretty lengthy. maybe `githubOpenPRBot` would be better?